### ISSUE TITLE
Support html checks

### ISSUE THREAD

**Issue by trevorsandy** [CJCombrink/SpellChecker-Plugin#93]
Check strings inside open and closed chevrons '>', '<' to enable html checking.

**Comment by CJCombrink** [comment]
@trevorsandy Can you please provide a bit more on what you are trying to achieve and examples.

The plugin only checks C/C++ header and source files, for comments and string literals. 
Thus if it is not in a .cpp or .h file (using the C++ code editor) it will not get checked.

**Comment by trevorsandy** [comment]
Very well. I understood this plugin to be "a spellchecker plugin for the Qt Creator IDE". 

I frequently edit content other than C++ source and header files in Qt Creator - for example, *.ui, *.qrc or actually any text-based file type. 

One file I frequently edit in Qt Creator is my release notes which is written in html (to link GitHub tickets).  

My thinking on creating this ticket is to suggest checking html content - perhaps the same way you are checking normal string literals. I'm not trying to tell you how to implement this capability but it seems like substituting '"' with open '>' and closing '<' chevrons could enable this.

As an example, consider the html line below...

`<tr><td class="tg-0lax"><a href="https://github.com/trevorsandy/lpub3d/issues/116">Change - General standardization #116</a></td></tr>`

The aim is to check the content between the chevrons >Change - General standardization #116<

Cheers,

**Comment by CJCombrink** [comment]
>  I understood this plugin to be "a spellchecker plugin for the Qt Creator IDE".
This is/was the goal when I started with the plugin, thus the layout of the files/project (technical info). 

At the start it handled much more files, but caused a lot of issues (crashes, bugs, weird behavior, etc) and I decided to restrict the C++ checker part very hard to C++. I write C++ code 99% of the time, so unfortunately I focus the effort there.
I am planning a project in QML, and will then most probably try to spend the effort to get that part implemented, but not sure when in future this will be.

The code should allow one to add checkers for other files/types, given the editor can provide the correct information, like comments (perhaps the Language Server plugin that is coming in QtC 4.8 might help a lot, but TBH I have no idea what it is, but sounds interesting)

Unfortunately HTM is not high on my list of priorities, but there might be some improvement in future, like a fallback for some type of files
